exercises/rational-numbers/canonical-data.json

```diff
@@ -59,16 +59,16 @@
           "cases": [
             {
               "description": "Subtract two positive rational numbers",
-              "property": "sub",
+              "property": "subtract",
```

Comment:
> Per #1603 this updates some properties to full words instead of abbreviations.

As the argument in #1603 is "`abs()` is a builtin in some environments", I should point out that `sub` is a reserved word in Perl, and `subtract` is a builtin in Haskell. As I pointed out in #1603, the premise to avoid keywords in all possible languages is neither reasonable nor necessary.